dashboard/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:latest
+
+WORKDIR /app
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+EXPOSE 8501
+
+CMD ["streamlit", "run", "dashboard.py", "--server.port=8501", "--server.address=0.0.0.0"]
```

Comment:
Does server address need to be specified here? Does it not mean localhost, does this still work when deployed with a public IP?

Reply:
Yes, we discussed this and came to an agreement as this server address allows you to access it from any address, localhost or other. It does work as it's currently deployed